src/PythonMigrationViewExtension/MigrationAssistant/migrate_2to3.py

```diff
@@ -5,12 +5,17 @@
 from pathlib import Path
 import sys
 
+
 def transform(source):
     python_zip_path = str(Path(os.path.dirname(os.__file__)).parents[0])
     python_zip_file = get_zip_file(python_zip_path)
     python_zip_folder = zipfile.ZipFile(os.path.join(python_zip_path, python_zip_file))
     fixers = get_all_fixers_from_zipfolder(python_zip_folder)
 
+    dir_path = os.getcwd()
+    sys.path.append(os.path.join(dir_path, '..\\..\\..\\src\\PythonMigrationViewExtension\\MigrationAssistant\\custom_fixers'))
```

Comment:
We can't reference the `src` folder. Copying the fixer to output and referencing that path should work.

Reply:
Yes, I was going to change this. I set this just for testing on my dev setup.